ipinfo/handler_utils.py

```diff
@@ -44,14 +44,19 @@
 BATCH_REQ_TIMEOUT_DEFAULT = 5
 
 
-def get_headers(access_token):
-    """Build headers for request to IPinfo API."""
-    headers = {
+def get_headers(access_token, custom_headers):
+    headers = {}
+
+    if custom_headers:
+        headers = custom_headers
```

Comment:
The way we should probably do it is, still keep our own headers, but then do a update of the dictionary using this one. Kind of like this in js:

```
d1 = {a: 1, b: 2}
d2 = {a: 3}
d3 = {...d1, ...d2}
```